contracts/season3/TalentSponsorship.sol

```diff
@@ -50,14 +54,20 @@ contract TalentSponsorship {
         string symbol
     );
 
-    constructor() {
+    constructor(address contractOwner) {
         totalSponsorships = 0;
         totalRevokedSponsorships = 0;
         totalClaimedSponsorships = 0;
+        transferOwnership(contractOwner);
+    }
+
+    modifier notDisabled() {
```

Comment:
I suggest using `enabled` or even `onlyWhileEnabled`. I think it's easier instead of having to think in the negative

Reply:
agree